### ISSUE TITLE
Remote control Back Button WebOS

### ISSUE THREAD

**Issue by tiorobby** [interfaced/zombiebox-platform-webos#3]
Hello... 
time ago I remember if I want to activate the function of raise the LG WebOs Menu when I reach the end of the "back History" I had to write a code line for that.

/**
	 * @override
	 */
	_backOnEmptyHistory() {
		return this.isDeviceWebos() ? this.device.showAppsManager() : this.device.exit();
	}

Is that necessary in the new Zombie Box versions?

I ask that coz I can't find that text on the documentation. Sorry if I'm making a mistake.
Best Rgds

ps. I'm very pleased with the Zombiebox release of my tv app so I say thanks for all your help.

**Comment by l1bbcsg** [comment]
Yes, this is still the case and was for quite a while, there were no recent changes in this behaviour. 
The method you want to use depends on desired UX scenario.

`showAppsManager` and `exit` do different things. The former raises applications menu while the latter kills the app completely which is more consistent with other platforms which is why it's the default behaviour in ZombieBox.
webOS UX checklist suggests showing app drawer is the preferred behaviour, but is not quite crystal clear on how the complete exit scenario should be executed, so it's often resolved in 1:1 QA in sellers lounge during app publishing unfortunately.

I think it might make sense to change the default behaviour of `_backOnEmptyHistory` on webOS actually, but more thought should be put into it.

**Comment by pedrosilvabos** [comment]
Hello, Im sorry for coming to this particular issue. @tiorobby  @l1bbcsg  @reeFridge  @nomean42  I have a DASH + Widevine stream and I am trying to play it, could someone give me some help ?
